signNverify.rst

```diff
@@ -380,24 +385,41 @@ specifying ``--group-id`` can also verify the container:
 PEM Key / X.509 Certificate Support
 ***********************************
 
-{Project} also supports signing SIF container images
-using a PEM format private key, and verifying with a PEM format public key, or
-X.509 certificate. Non-PGP signatures are implemented using the `Dead Simple
-Signing Envelope <https://github.com/secure-systems-lab/dsse>`__ (DSSE)
-standard.
+{Project} also supports signing SIF container images using a PEM format private
+key, and verifying with a PEM format public key, or X.509 certificate. Non-PGP
+signatures are implemented using the `Dead Simple Signing Envelope
+<https://github.com/secure-systems-lab/dsse>`__ (DSSE) standard.
+
+The {Project} GitHub repo contains keys and certificates useful for testing. If
+you want to use them to carry out the commands below, first, carry out the
+following commands:
+
+.. code::
+
+   $ git clone https://github.com/apptainer/apptainer.git
+
+   $ export KEYD="${PWD}/apptainer/test/keys"
+
+   $ export CERTD="${PWD}/apptainer/test/certs"
+
+Information on creating PEM files can be found in the `encrypted container docs
+<https://apptainer.org/docs/user/main/encryption.html#pem-file-encryption>`__,
+and the method for creating certificates is documented `here
+<https://github.com/apptainer/apptainer/blob/main/test/certs/gen_certs.go>`__.
```

Comment:
In place of `apptainer/apptainer` use `{orgrepo}` and in place of `main` use `{repobranch}`.